src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxNumber = 100;

```

Comment:
naming convention:  [this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)